helm/ako/values.yaml

```diff
@@ -29,12 +29,14 @@ AKOSettings:
   servicesAPI: false # Flag that enables AKO in services API mode: https://kubernetes-sigs.github.io/service-apis/. Currently implemented only for L4. This flag uses the upstream GA APIs which are not backward compatible 
                      # with the advancedL4 APIs which uses a fork and a version of v1alpha1pre1 
   vipPerNamespace: "false" # Enabling this flag would tell AKO to create Parent VS per Namespace in EVH mode
+  istioEnabled: "false"
```

Comment:
comment on what this flag does will be helpful here

Reply:
added comment